README.md

```diff
@@ -1,19 +1,61 @@
-# Experimental Design Library
+experimental-design-lib
+-----------------------------------
+# <p align=center>Experimental Design Library</p>
```

Comment:
You only need one centered `div`.
https://github.com/qbicsoftware/sample-tracking-service/blob/master/README.md